signalsigterm.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// +build darwin linux windows unix
```

Comment:
is unix a recognized build tag? i thought this would need to explicitly add each unix OS.